guides/7-jlp/1-JLP.md

```diff
@@ -9,41 +9,57 @@ title: "JLP Overview"
+
+
+### How to Become a Liquidity Provider (LP)
+
+Anyone can become a Liquidity Provider by contributing assets or tokens to the Jupiter Liquidity Provider Pool (JLP Pool).
  - JLP tokens represent your share of the pool.
  - Any asset traded on JUP can be used to buy into JLP.
  - There are fees associated with buying into the JLP pool (see [Target Ratio and Fees](../jlp/How-JLP-Works#target-ratio-and-fees)).
- - More on how to get the token in [How To Get JLP](../jlp/How-To-Get-JLP).
 
-### Rewards and Earnings:
+More details on how to get the token in [How To Get JLP](../jlp/How-To-Get-JLP).
 
- - 75% of all perpetual trading fees are deposited into the JLP pool to be distributed among JLP holders (reflected as an increase in the JLP token price).
- - Forms an Annual Percentage Rate (APR).
- - Higher trading volume generally leads to higher earnings for JLP holders.
- - More on how it works in [How JLP Works](../jlp/How-JLP-Works).
+:::tip
+The best way to purchase or exit JLP is always via [Jupiter Swap](https://jup.ag/swap/USDC-JLP).
+:::
 
-### Advantages of the JLP System:
+ ### Advantages of the JLP System
 
- - No need to actively "stake" tokens or "harvest" yields.
- - APR earnings are embedded within each JLP token and accumulate automatically.
- - The JLP token is an SPL token, making it easily transferable and tradable like other SPL tokens within the Solana ecosystem.
- - AMM pools can be established for trading JLP tokens.
+ The JLP system offers a user-friendly method for participants to earn passive income while contributing to the liquidity and stability of the trading environment:
 
+- LPs do not need to actively "stake" tokens or "harvest" yields - APR earnings are embedded within each JLP token and accumulate automatically (reflected as an increase in the JLP token price).
+- The JLP token is also an SPL token, making it easily transferable and tradable like other SPL tokens within the Solana ecosystem.
+- AMM pools can be established for trading JLP tokens.
 
-Overall, the JLP system offers a user-friendly method for participants to earn passive income while contributing to the liquidity and stability of the trading environment.
+:::info
+JLP is a liquidity token for the Jupiter Perpetuals platform, and JUP is the governance token of the JUP DAO.
+:::
 
-**Note**: JLP is a liquidity token for the Jupiter Perpetuals platform, and JUP is the governance token of the JUP DAO.
+### Rewards and Earnings
 
-#### The best way to purchase or exit JLP is always via [Jupiter Swap](https://jup.ag/swap/USDC-JLP).
+The **APY** of JLP is calculated based on 75% of fees generated from perps trading activities (not including assets appreciation and traders PnL). 
+
+The generated fees are distributed back to liquidity providers by redepositing the fees into the pool hourly.
```

Comment:
```suggestion
The generated fees are distributed back to liquidity providers
```

We actually distribute fees back to the pool once a week (which is when we calculate the APR of the pool), so we can change that here